pkg/minikube/cluster/commands.go

```diff
@@ -16,10 +16,33 @@ limitations under the License.
 
 package cluster
 
-var startCommand = `
-# Run with nohup so it stays up. Redirect logs to useful places.
-PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start --generate-certs=false > /var/log/localkube.out 2> /var/log/localkube.err < /dev/null &
-`
+import (
+	gflag "flag"
+	"fmt"
+
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+const (
+	remoteLocalKubeErrPath = "/var/log/localkube.err"
+	remoteLocalKubeOutPath = "/var/log/localkube.out"
+)
 
 // Kill any running instances.
 var stopCommand = "killall localkube | true"
+
+func GetStartCommand() string {
```

Comment:
Also could you add a few unit tests for this function? You might have to change this function so the flags are passed in instead of getting them from the global flag package.